### PR TITLE
Ngath 1105 and Vland 1200 

### PR DESCRIPTION
**Pull request by bobbyjim**
Added a proper header to Ngathksirz sector.

It renders okay in the poster maker, so it should be OK, right??

This MR also has edits to the Vland sector for 1120, to conform it to MegaTraveller GDW sources better.

### CODE REVIEW

File: res/Sectors/M1120/1120_Vlan.xml [hunk 1]
```diff
@@ -43,11 +43,12 @@
...
+    <Border Allegiance="LuIm" WrapLabel="true" LabelPosition="3133" Color="DarkRed">3029 3129 3128 3228 3229 3230 3231 3232 3233 3234 3235 3236 3237 3238 3239 3140 3139 3138 3137 3136 3035 3135 3134 3133 3132 3131 3130 3029 </Border>
+    <Border Allegiance="StIm" LabelPosition="1838" Color="Crimson">0036 0137 0237 0338 0339 0340 0341 0241 0141 0041 0040 0039 0038 0037 0036 </Border>
+    <Border Allegiance="V17D" LabelPosition="2801" Color="OliveDrab">2602 2703 2803 2903 3002 3102 3201 3202 3103 3003 3004 2905 2804 2705 2605 2604 2603 2602 </Border>
+    <Border Allegiance="VNgC" LabelPosition="1502" Color="OliveDrab">0402 0502 0601 0701 0801 0902 1001 1102 1201 1202 1303 1403 1303 1203 1104 1004 1003 0903 0802 0702 0601 0502 0402 </Border>
+    <Border Allegiance="VNgC" LabelPosition="1502" Color="OliveDrab">1701 1801 1901 2001 2101 2001 1902 1903 1904 1804 1705 1704 1703 1702 1701 1600 </Border>
```
Comment (inexorabletash) [inline res/Sectors/M1120/1120_Vlan.xml:50]: This has the same label position as the previous border. Mistake?

File: res/Sectors/M1105/Ngathksirz.sec [hunk 2]
```diff
@@ -1,8 +1,37 @@
-Ngathksirz
-6 1
-                               1    2         3 4              5 6   7  8
-#
+# Generated by https://travellermap.com
```
Comment (inexorabletash) [inline res/Sectors/M1105/Ngathksirz.sec:1]: This would just be blown away - please don't include it.

File: res/Sectors/M1105/Ngathksirz.sec [hunk 3]
```diff
@@ -18,9 +47,7 @@ Ngathksirz
 #Alleg: 13: Vb: "Vargr (Baknae Alliance)"
 #Alleg: 14: V5: "Vargr (Ruler of Five)"
 #Alleg: 15: Vd: "Vargr (Democracy of Greats)"
-#
-#
-#Created by John G. Wood <john@elvwood.org>
+
 Hex  Name                             UWP       Remarks                {Ix} (Ex)    [Cx]   N B  Z  PBG  W A      Stellar
```
Comment (inexorabletash) [inline res/Sectors/M1105/Ngathksirz.sec:51]: Hmmm... this looks like it's not .SEC, but SecondSurvey. I'm kinda shocked it works... I guess if Type isn't specified the code guesses?

Can you update the PR to include a fix to the M1105 XML too?

**Review by inexorabletash — CHANGES_REQUESTED** [review]
In terms of reviews, it would help if the changes were clustered, e.g. PRs for one sector across milieux, or multiple sectors in a single milieu.

**Comment by bobbyjim** [comment]
I think I have to re-do those changes.

**Pull request closed without merging**